src/frontend/types/AppData.ts

```diff
@@ -24,9 +25,8 @@ export enum appNames {
   MARKDOWN = 'markdown',
 }
 
-export interface AppData {
+export interface AppConfig {
```

Comment:
Maybe you should also change the file name ?

Reply:
AppData is still exported:
```
export type AppData = AppConfig & AuthenticatedUser;
```